### ISSUE TITLE
Function to convert between IPA and Arpabet phonemes

### ISSUE THREAD

**Issue by dhowe** [dhowe/RiTa#208]
include functions to convert back and forth between IPA and Arpabet phonemes?

**Comment by dhowe** [comment]
@CyrusSUEN status?

**Comment by dhowe** [comment]
@CyrusSUEN status?

**Comment by dhowe** [comment]
@cqx931 when this is ready, I'd like to use it in our FeatureAnalysis example

@CyrusSUEN status (please update)?

**Comment by CyrusSUEN** [comment]
> status

The syllables related work are in `Phoneme.java`, `PhonemeTest.java` and `Constants.java` ([which](https://github.com/dhowe/RiTa/commit/3f4a702de69bd926e3f63cc1581031135e59b3ea#diff-16e967d8227489b2c991d7ea1989b7b1) includes type 0, 1 and 2 stressed arpabet syllables) 

Currently we can do `Phoneme.arpaToIPA()`arpabet to IPA but not the other way around.
<img width="289" alt="screen shot 2017-04-16 at 8 20 05 pm" src="https://cloud.githubusercontent.com/assets/2461812/25071149/aa8e92f0-22e2-11e7-9c3d-732eb371ea18.png">

I've given @cqx931 an introduction to syllables and I think she is able to pick up my work. 👍

**Comment by CyrusSUEN** [comment]
Looks like after implementing `Phoneme.IPAToArpa()` this ticket can be closed.

**Comment by dhowe** [comment]
Thanks, @CyrusSUEN. 
Actually, the first step is to implement as arpaToIPA as a separate file in js, but only after higher-priority tickets are finished

**Comment by cqx931** [comment]
Please check the first step here - implement arpaToIPA() as a separate file in js
https://github.com/dhowe/RiTaJS/pull/71

**Comment by dhowe** [comment]
Next step is to integrate this into a new version of the Analysis example

**Comment by cqx931** [comment]
Supporting IPA styles phonemes in analyze()?

**Comment by dhowe** [comment]
No, just for display (see above)

**Comment by cqx931** [comment]
Replace the phonemes below with IPA syllables while every other things stay the same?
![image](https://cloud.githubusercontent.com/assets/3056154/26445059/45ca6242-4171-11e7-9e50-e3ec12917464.png)

**Comment by dhowe** [comment]
No, show the part of speech and IPA phonemes below the word, exactly as in the image above (though only one pronunciation). Leave phonemes in bubble the same for now.

**Comment by dhowe** [comment]
another visual option:

![image](https://cloud.githubusercontent.com/assets/737638/26449654/ff07e194-4185-11e7-9ca7-d4e8021e9ca6.png)

**Comment by cqx931** [comment]
Please check: https://github.com/dhowe/RiTaJS/pulls, if this is ok, I will update the rita website as well.

**Comment by cqx931** [comment]
Please check: https://github.com/dhowe/RiTaJS/pull/73
If this is ok, I will update it in RiTa website as well

**Comment by dhowe** [comment]
status?
